src/components/VxpAccordion.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <StackLayout class="accordion">
+    <slot></slot>
+  </StackLayout>
+</template>
+
+<script>
+import StackLayout from '../layouts/StackLayout';
+
+export default {
+  name: 'VxpAccordion',
+  components: { StackLayout },
+  props: {
+    initialExpandedItems: {
+      required: false,
+      type: Array,
+      default: function() {
+        return [];
+      },
+      validator: items => Array.isArray(items) && !items.some(n => typeof n !== 'number'),
+    },
+    autoCollapse: {
+      required: false,
+      type: Boolean,
+      default: false,
+    },
+  },
+  mounted() {
+    this.$on('child-registered', child => {
```

Comment:
we should avoid anonymus functions. I think it is a better implementation to keep event handlers under methods key

Reply:
anonymus functions replaced with predefined methods.